client/starwhale/api/_impl/dataset/loader.py

```diff
@@ -92,6 +94,44 @@ def _do_iter_row(self) -> t.Generator[TabularDatasetRow, None, None]:
             for row in self.tabular_dataset.scan():
                 yield row
         else:
+
+            def _travel_link(obj: t.Any) -> t.List[Link]:
```

Comment:
move `_travel_link` and `sign_uris` out of `_do_iter_row` function? it may be clear for the code organization.

Reply:
done